official-build.props

```
@@ -0,0 +1 @@
+SEPG_BUILD_ENV_IMAGE=cafapi/buildenv-jdk17:1.5.1
```

Update to the latest:

```suggestion
SEPG_BUILD_ENV_IMAGE=cafapi/buildenv-jdk17:4.1.0
```